### ISSUE TITLE
CSharpErrorLog.xml is broken in master.

### ISSUE THREAD

**Issue by CyrusNajmabadi** [dotnet/roslyn#17686]

**Comment by mavasani** [comment]
Investigated this further and it seems the issue here is that FSA is broken in master (works fine on Dev15 RTM).

Manual Repro:

1. Unzip https://github.com/dotnet/roslyn-internal/blob/master/Closed/Test/Solutions/Newtonsoft.Json.zip
2. Open VS and enable full solution analysis
3. Open Newtonsoft.Json.Portable.sln and wait for error list to complete (no ...)

**Expected:** Around 9k stylecop diagnostics
**Actual:** No diagnostics

4. Open any source file in solution - you will now start seeing diagnostics in that file.

Basically, it seems only open file diagnostics are getting shown.

@heejaechang I am assigning this to you.

**Comment by heejaechang** [comment]
@jasonmalinowski @dotnet/roslyn-ide does workspace eventing has changed? FSA isn't running since it doesn't get solution added event until a file is opened. brief debugging shows that, until a file is opened, VSWorkspace is actually empty.

is this some new change?

**Comment by heejaechang** [comment]
I think it is IDE bug, IDE should start pushing workspace event once it got solution fully loaded event from VS.

**Comment by heejaechang** [comment]
@Pilchie I gave this bug back to IDE.

**Comment by jasonmalinowski** [comment]
Ah, this is the bug @Pilchie fixed. For some reason we haven't merged #17618 yet so it's still (!) not in master. Following up with that now...

**Comment by jasonmalinowski** [comment]
Oh this is the bug @Pilchie fixed. It only flowed into master as a part of #17654.
